README.md

```diff
@@ -192,6 +193,35 @@ with the exception of dynamic tuning functionality provided by some of the plugi
 
 The following Tuned plugins are currently not supported:
 
-* bootloader
```

Comment:
Why is this being removed in a PR that doesn't implement the `bootloader` plugin?

Reply:
@jmencak  It looks correct since this is removed from the "**not** supported" list.

Reply:
This PR is really a mixture of two PRs and should be split. A) Support for the `bootloader` plugin, B) possibly add an openshift realtime profile as necessary.  I haven't verified this PR actually includes full `bootloader` plugin support.  If this is true, then yes, I agree with this change, but my point to split this PR into two still stands.

Reply:
Ok i left it on hold until the situation is clarified. Tomorrow i will amend this patch to just create a realtime profile without the bootloader part.

Reply:
Right.  The default tuned CR that you're modifying with this PR tries to deliver default out-of-the box tuning for OpenShift control plane and nodes.  The fact it ships 4 profiles (2x tuning for ES on control plane and nodes) is probably already a mistake/bug/issue and should be addressed.  The default tuning should addresses day-1 operations and anything outside day-1 operations (profiles for specific applications, configuring hugepages, sap, nfv, oracle, realtime profiles, ...) should really be set by cluster admins as a day-2 operation.  Adding a realtime profile like that would also only add duplication to what `tuned-profiles-realtime` package ships.  The path forward in my opinion is to add additional `tuned-profiles-*` packages as the operator support for all the tuned plugins improves.

Reply:
/cc @zvonkok